k8s/sandbox/common/idam-sprod/idam-web-public.yaml

```
@@ -0,0 +1,28 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-web-public
+  namespace: idam-sprod
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:preview-*
+spec:
+  releaseName: idam-web-public
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-web-public
+    version: 0.2.3
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/web-public:preview
+      ingressHost: idam-web-public-aks.sprod.platform.hmcts.net
```

```suggestion
      ingressHost: idam-web-public-sprod.sandbox.platform.hmcts.net
```

This will require an additional ingress for Front Door https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/sbox.tfvars#L25-L73

---

Have changed to domain to idam-web-public-sprod-aks.sandbox.platform.hmcts.net as i cannot get hold of a wildcard sprod cert

---

added sprod to aks backends
https://github.com/hmcts/azure-platform-terraform/pull/154